tests/models/gpt2/test_gpt2.py

```diff
@@ -17,11 +17,20 @@ def test_gpt2(record_property):
     m = AutoModelForSequenceClassification.from_pretrained(
         model_name, torch_dtype=torch.bfloat16
     )
+    for param in m.parameters():
+        param.requires_grad = False
     m.eval()
 
     # Set up sample input
     test_input = "This is a sample text from "
-    inputs = tokenizer(test_input, return_tensors="pt")
+    inputs = tokenizer.encode_plus(
```

Comment:
can you clarify what happens here?

Reply:
In this case, `encode_plus` isn't any different since I'm not using the extra information like masking that `encode_plus` provides. I believe BERT was because the input had multiple sentences. I can revert this until the changes make sense.

Reply:
Since GPT-2 isn't working in this PR anyway, I reverted this change.